SQL/New_patches/2019-12-17-RemovePasswordExpiry.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE users DROP COLUMN Password_expiry;
+ALTER TABLE users ADD COLUMN Password_expired TINYINT(1) NOT NULL DEFAULT 0;
```

Comment:
```suggestion
ALTER TABLE users ADD COLUMN Password_expired TINYINT(1) NOT NULL DEFAULT 0 AFTER Password_hash;
```
to keep the column same order.